### DISCUSSION TITLE
supporting column-level

### DISCUSSION THREAD

**Discussion by javaht** [reata/sqllineage4j#4] [General]
Whether to consider supporting column-level data consanguinity？ when

**Comment by reata** [comment]
It's a long term commitment to catch up with sqllineage main repo, implementing all features in sqllineage4j, including column level lineage. But given current circumstances, more of my focus will be on main repo for the rest of 2023 as there're a lot of unfinished work there. 

Contribution is more than welcome if you're interested.
